docs/setup/pip.md

```diff
@@ -13,25 +16,25 @@ We support :simple-python: **Python from 3.8 to 3.12**.
     sudo apt-get update -y && sudo apt-get install wget lsb-release -y && \
     CODENAME=$(lsb_release -cs) && \
     TEMP_DEB="$(mktemp)" && \
-    wget -O "$TEMP_DEB" "https://github.com/KhiopsML/khiops/releases/download/10.2.3/khiops-core-openmpi_10.2.3-1-${CODENAME}.amd64.deb" && \
+    wget -O "$TEMP_DEB" "https://github.com/KhiopsML/khiops/releases/download/{{ KHIOPS_VERSION }}/khiops-core-openmpi_{{ KHIOPS_VERSION }}-1-${CODENAME}.amd64.deb" && \
     sudo dpkg -i "$TEMP_DEB" || sudo apt-get -f -y install && \
     rm -f $TEMP_DEB && \
-    pip install 'https://github.com/KhiopsML/khiops-python/releases/download/10.2.3.0/khiops-10.2.3.0.tar.gz'
+    pip install 'https://github.com/KhiopsML/khiops-python/releases/download/{{ TARGZ_KHIOPS_PYTHON_VERSION }}/khiops-{{ KHIOPS_PYTHON_VERSION }}.tar.gz'
```

Comment:
It's the other way round: in the release URL (_except_ the tar.gz), we need to keep the `KHIOPS_PYTHON_VERSION`; the `TARGZ_KHIOPS_PYTHON_VERSION` needs to be put in the tar.gz file.

Whence:

```sh
pip install 'https://github.com/KhiopsML/khiops-python/releases/download/{{ KHIOPS_PYTHON_VERSION }}/khiops-{{ TARGZ_KHIOPS_PYTHON_VERSION }}.tar.gz'
```